src/OpenTelemetry.ClrProfiler.Managed/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("IntegrationTests.Helpers")]
+[assembly: InternalsVisibleTo("IntegrationTests.MongoDB")]
```

Comment:
why is it needed? maybe it would be better to hardcode something or make it public?

Reply:
Seems leftover. I guess I was reusing some utils from that dll, and did not want to create separate utils dll just yet.